automod/engine/context.go

```diff
@@ -52,8 +52,7 @@ type RecordOp struct {
 	Collection syntax.NSID
 	RecordKey  syntax.RecordKey
 	CID        *syntax.CID
-	// NOTE: usually a *pointer*, not the value itself
-	Value any
+	RecordCBOR *[]byte
```

Comment:
This can just be `[]byte` rather than `*[]byte`.

I'm guessing the extra pointer appeared out of a habit of using pointers as a stand-in for `Optional<T>`, but... It's not gopheric.  :)

(Okay, admittedly, making distinctions between `nil` and `[]T{}` is also considered non-gopheric.  But in cases where one really wants to use `nil` as a distinction, I think having fewer unnecessary pointers is the lesser of evils.)

(The only time one needs a `*[]T` is if one needs to be able to replace the variable's content with a whole new slice allocation, and _has no other handle_ to it.  Sometimes, fairly rarely, one ends up with a function that takes a `*[]T` parameter because it wants to be able to grow that slice (without having an API involving returning a new one and leaving the caller to handle the assignment).  But if the thing is in a struct, like this... you _already_ have another updatable handle to it, so that situation doesn't apply.  (And since this is supposed to be "immutable", it doubly doesn't apply!))

Reply:
In naming nits: I'd maybe put "raw" or "original" somewhere in the var name, so the reader doesn't have to do the mental walk through "ah, CBOR, since that's a serial format this probably means this is unparsed original raw bytes".  But that's just a (very mild) opinion.

Reply:
cool, I switched to `[]byte`. I don't love any "raw" or "original" variants so i'm going to stick with `RecordCBOR` for now.